.gitignore

```diff
@@ -1,5 +1,4 @@
 .sass-cache
```

Comment:
Not sure if this is generated by compass or not. If so … should be removed as well.